.github/workflows/gh-page.yml

```diff
@@ -0,0 +1,35 @@
+name: main
+
+on:
+  push:
+    branches:
+      - "development"
+      - "add-obsolete-action-command"
+  
+jobs:
+  publish-gh-pages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install Dependency
+      working-directory: ./Dashboard
+      run: yarn
+
+    - name: Build .env
+      run: |
+        echo "VITE_OBSOLETE_DATA_PATH=$VITE_OBSOLETE_DATA_PATH" >> ./Dashboard/.env.production
+        echo "VITE_API_PATH=$VITE_API_PATH" >> ./Dashboard/.env.production
+
+    - name: Build Dashboard
+      working-directory: ./Dashboard
+      run: yarn build --outDir ../publish
+
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./publish
+        destination_dir: ${{ github.ref_name }}
```

Comment:
It seems you intended that the *Obsolete Dashboard*'s url becomes `https://planetarium.github.io/lib9c/development`. RIght?

Reply:
It should be `https://planetarium.github.io/lib9c/dashboard`. Thank you for let me know.